config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml

```diff
@@ -1144,6 +1144,7 @@ spec:
                           description: Device name
                           type: string
                         encrypted:
+                          default: true
```

Comment:
All this defaulting needs to go by removing the kubebuilder default and running `make generate`